docs/docs/usage/sbt-projects.md

```diff
@@ -3,10 +3,4 @@ layout: doc-page
 title: "Using Dotty with sbt"
 ---
 
-It is now possible to use Dotty with sbt thanks to the dotty-bridge project.
-There are two alternatives in how to create an sbt project that uses dotty:
-
-* [dotty-example-project](https://github.com/smarter/dotty-example-project)
-  for a simple sbt project that compiles code with Dotty
-* [sbt-dotty](https://github.com/felixmulder/sbt-dotty) an sbt plugin that
-  takes care of all dependencies and settings needed to get a Dotty sbt project
+See the [example project](https://github.com/lampepfl/dotty-example-project).
```

Comment:
We could just put the config info from the front-page into an `_include` and as such we don't need to duplicate any info here.